tests/unit/models/v0.2/modelResponse.test.js

```diff
@@ -43,53 +43,45 @@ const ilsResponse = {
 };
 
 const patronCreatorResult = {
-  data: [
```

Comment:
We are intentionally removing the containing array so that we only return one patron record.